packages/www/components/Dashboard/AssetDetails/AssetPlayerBox/ProcessingProgress.tsx

```diff
@@ -0,0 +1,13 @@
+import { useMemo } from "react";
+import { Asset } from "livepeer";
+import Progress from "./Progress";
+
+const ProcessingProgress = ({ asset }: { asset?: Asset }) => {
```

Comment:
For the updating frequency on the list, WDYT of allowing this component to receive an `id` instead of the full `asset`? Then if it receives an `id` instead, it could fetch the asset itself with a shorter refetch interval, instead of depending on the `15s` refetch for the list (which I believe is because of the cost, yeah).

This is optional though, don't worry about it if it's more than trivial to implement.

Reply:
This would be relatively easy to do. But this component is the one that shows the progress in the Asset details page, where the refresh interval is 5s. So the problem is not here.

The problem is in the Asset lists, where a Cell component displays the info passed. There would be against the pattern make a Cell component fetch state. They should just rended data, because the parent Table component is already doing a bunch of performance optimisations and the parent is the one in charge of managing the data.

Reply:
I see, makes a lot of sense. Thanks for explaining the context.